packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -55,7 +55,7 @@ export class CreateAccountStore {
    * Reinitialize account information
    */
   @action
-  clear = async () => {
+  clear = () => {
```

Comment:
hmm, the CI shows `fether-react: PASS src/stores/createAccountStore.spec.js`

Reply:
**correction** 

it shouldn't run fether-react/src/utils/testHelpers/storeTests.js anyway, as it's not a `.spec.js` file.

i'm using yarn v1.15.2.

if i use Node.js v10.15.3 (greater than minimum version of >=10.10.0 defined in the package.json) and run `yarn; yarn test`, it runs all the test files **except** for the file:
fether-react/src/stores/createAccountStore.spec.js

but if i switch to a newer version of Node.js v11.13.0, and run `yarn; yarn test`, then all the test files run ok.

also, i noticed that the fether-react tests didn't have colours enabled, so i've pushed a commit to enable them (i haven't enabled `--coverage` as the list is a bit too long)

note that i also tried using jest instead of craco for running the fether-react tests in a separate branch, but it wasn't necessary https://github.com/paritytech/fether/commit/7d1aa6a64223ea67ff2a521b23f263f572594bfa

Reply:
i just tried using Node.js v11.0.0 and all the tests ran, and then switched back to Node.js v10.15.3 again, but this time all the tests ran (i.e. i deliberately made a test fail in createAccountStore.spec.js and it triggered).

**so no further action required**

Reply:
@axelchalon sorry for writing so much, i thought there was an issue with running the tests but there isn't one. so i've addressed your comment and enabled colours for the fether-react tests